code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -210,10 +210,11 @@
 	if(prob(33))
 		M.adjustBruteLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
 		M.adjustFireLoss(-2*REAGENTS_EFFECT_MULTIPLIER)
-	if(ishuman(M))
-		var/mob/living/carbon/human/H = M
-		if(!H.species.exotic_blood && !(H.species.flags & NO_BLOOD) && prob(33))
-			H.vessel.add_reagent("blood", 1)
+		if(ishuman(M))
```

Comment:
This should be it's own separate 33% probability check; not on the same one as the healing one.